app/abilities/curriculum-inventory.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
```

Comment:
you are correct, sir. this is kruft, and i will remove this. thanks for catching that.